### PR TITLE
Hotfix for avatars uploading in settings view

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
- UseRequest appears to be not working with avatars cropper, so need to create a quick hotfix for it
- Polishing of settings views

image

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Profile.kt [hunk 1]
```diff
@@ -45,8 +57,25 @@ val profileSettingsCard: FC<SettingsProps> = FC { props ->
             avatarWindowOpen.closeWindow()
         }
         imageUpload = { file ->
-            useRequest {
-                postImageUpload(file, props.userInfo?.name, AvatarType.USER, ::loadingHandler)
+            // FixMe: we are inside of FC here, but without any reason avatarForm component is ignoring
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Profile.kt:60]: @sanyavertolet can you please investigate it if you will have time?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Profile.kt [hunk 2]
```diff
@@ -45,8 +57,25 @@ val profileSettingsCard: FC<SettingsProps> = FC { props ->
...
+                val response = request(
+                    url = "$apiUrl/avatar/upload".withParams(jso<dynamic> {
+                        owner = props.userInfo?.name
+                        this.type = AvatarType.USER
+                    }),
+                    method = "POST",
+                    headers = Headers().apply { append(CONTENT_LENGTH_CUSTOM, file.size.toString()) },
+                    body = FormData().apply { set(FILE_PART_NAME, file) },
+                    loadingHandler = ::noopLoadingHandler,
+                    responseHandler = ::noopResponseHandler,
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Profile.kt:74]: and please add normal responseHandler here (with modal window)

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Profile.kt [hunk 3]
```diff
@@ -142,36 +168,32 @@ val profileSettingsCard: FC<SettingsProps> = FC { props ->
...
-    placeholder: String,
 ) {
-    label {
-        className = ClassName("btn animated-provider")
+    Link {
+        className = ClassName("btn px-0 pt-0")
         title = AVATAR_TITLE
         onClick = {
             avatarWindowOpen.openWindow()
         }
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Profile.kt:181]: Are you sure you want to use `Link` with `onClick` handler? Better use button for this purpose.

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/RequestUtils.kt
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/RequestUtils.kt:1]: What was the problem with the file? Was diktat disabled here?

**Pull request merged**